turbinia/e2e/setup-gce.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+# This script will
+# - setup a GCE Ubuntu 18.04 LTS base image for Turbinia e2e testing.
```

Comment:
It configures a GCE instance (and project infra) to be ready to run the Turbinia end-to-end tests. In our setup Jenkins takes care of creating the actual GCE instance and this script is run in the GCE instance to configure it. It can be run outside of Jenkins in any GCE instance that you would like to use for end-to-end testing Turbinia.

I'll clarify that a bit better in the script comment.

Reply:
Ah okay got it, I was thinking the script was doing that, but makes more sense.